db/migrate/20240531160738_add_status_group_to_user.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddStatusGroupToUser < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :status_group, :integer, null: false, limit: 2, default: 0, comment: 'Used as enum in Rails'
```

Comment:
This could also be "limit: 1", right?

Reply:
Yes, indeed. I changed it here (and in the `schema.rb`).